post/OpenSeesPost.dpr

```diff
@@ -3389,39 +3404,54 @@ begin
 
         WriteResults(false,
                      false,'Reading ShellDKGQ stresses (membrane) for layer '+IntToStr(i)+' ',
-                           'ShellDKGQ_stress_Layer'+IntToStr(i),'ShellDKGQ','Stresses-Membrane (L'+IntToStr(i)+')',
+                           'ShellDKGQ_concrete_stress_Layer'+IntToStr(i),'ShellDKGQ','Stresses-Membrane (L'+IntToStr(i)+')',
                            's11','s22','s12','','','','kPa',4,5,true,0,1,2,-1,-1,-1);
 
         WriteResults(false,
                      false,'Reading ShellDKGQ stresses (shear) for layer '+IntToStr(i)+' ',
-                           'ShellDKGQ_stress_Layer'+IntToStr(i),'ShellDKGQ','Stresses-Shear (L'+IntToStr(i)+')',
+                           'ShellDKGQ_concrete_stress_Layer'+IntToStr(i),'ShellDKGQ','Stresses-Shear (L'+IntToStr(i)+')',
                            't1','t2','N/A','','','','kPa',4,5,true,3,4,-1,-1,-1,-1);
 
         WriteResults(false,
                      false,'Reading ShellDKGQ strains (membrane) for layer '+IntToStr(i)+' ',
-                           'ShellDKGQ_strain_Layer'+IntToStr(i),'ShellDKGQ','Strains-Membrane (L'+IntToStr(i)+')',
+                           'ShellDKGQ_concrete_strain_Layer'+IntToStr(i),'ShellDKGQ','Strains-Membrane (L'+IntToStr(i)+')',
                            'e11','e22','e12','','','','-',4,5,true,0,1,2,-1,-1,-1);
 
         WriteResults(false,
                      false,'Reading ShellDKGQ strains (shear) for layer '+IntToStr(i)+' ',
-                           'ShellDKGQ_strain_Layer'+IntToStr(i),'ShellDKGQ','Strains-Shear (L'+IntToStr(i)+')',
+                           'ShellDKGQ_concrete_strain_Layer'+IntToStr(i),'ShellDKGQ','Strains-Shear (L'+IntToStr(i)+')',
                            'g1','g2','N/A','','','','-',4,5,true,3,4,-1,-1,-1,-1);
 
+        WriteResults(false,
+                     false,'Reading ShellDKGQ Axial stresses for layer '+IntToStr(i)+' ',
+                           'ShellDKGQ_steel_stress_Layer'+IntToStr(i),'ShellDKGQ','Axial-Stresses (L'+IntToStr(i)+')',
```

Comment:
But the way you chose to name this as "Axial-Stresses" is a bit confusing (although technically correct) - May you kindly update this to be "Steel Layer Stresses" instead of "Axial-Stresses".

Reply:
Unless this is the way GiD recognizes that this is axial stress based on the name and using something else would confuse it. Double-check before you apply my comment.

Reply:
Your guess is right! 'Axial-Stresses' here can be changed to the name you suggested.